t/api/05-machines.t

```diff
@@ -163,6 +163,69 @@ is_deeply(
 $res = $t->delete_ok("/api/v1/machines/$machine_id")->status_is(200);
 $res = $t->delete_ok("/api/v1/machines/$machine_id")->status_is(404);    #not found
 
+subtest 'trim whitespace characters' => sub {
+    $t->post_ok(
+        '/api/v1/machines',
+        form => {
+            name                => " createwithspace ",
```

Comment:
would you mind making that more readable with underscore as word separators?

Reply:
I will modify those names